web/Dockerfile

```diff
@@ -8,9 +8,10 @@ COPY scripts/start-nodaemon.sh /usr/bin
 COPY scripts/mysqlwait.sh /usr/bin
 
 # python2.7 is required to run supervisor
-RUN apt-get update -y && apt-get install -y \
+RUN apt-get update -y && apt-get upgrade && \
```

Comment:
@VincentHokie  im getting a prompt when running the 'upgrade' step. does it need the "-y" flag?

Reply:
ah, must have missed this..adding

Reply:
weird, it doesn't prompt me..won't hurt to make the command more complete though